mesop/pip_package/requirements.txt

```diff
@@ -1,9 +1,7 @@
 absl-py
-deepdiff==6.7.1
+deepdiff
 flask
-libcst==1.1.0
 markdown
 protobuf
-pydantic==1.10.13 # --no-binary pydantic
```

Comment:
Do we still need pydantic in Mesop? In the earlier requirements.txt you left it in. I guess I'm unsure how what's the difference from  requirements.txt in pip_package and the other one.

Reply:
good catch! I shouldn't have deleted it.

Reply:
Re: the difference. This requirement.txt is what's actually used when people do `pip install mesop`. The one in build_defs is what we actually use when compiling Mesop w/ Bazel.

i've filed an issue: https://github.com/google/mesop/issues/467 to refactor this to avoid this duplication.